src/style.css

```diff
@@ -1,3 +1,72 @@
-body {
+:root {
+  --main-color: #00acdc;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__item {
+  margin-right: 20px;
```

Comment:
Be consistent with your vertical margins. It's recommended to add only top or only bottom margin, but not both, to prevent issues with margin collapse. In this case, consider using only `margin-bottom` for `.nav__item`.